### ISSUE TITLE
Invalid check for processor existence

### ISSUE THREAD

**Issue by xumix** [JimBobSquarePants/ImageProcessor#125]
ImageHelpers.cs
```
IWebGraphicsProcessor format = ImageProcessorConfiguration.Instance.GraphicsProcessors
.First(p => typeof(Format) == p.GetType());
if (format != null)
{
match = format.RegexPattern.Match(queryString);
}
```
This line will always throw an Exception if the module is not found
the proper check must be ```FirstOrDefault```

**Comment by JimBobSquarePants** [comment]
Pull request?

**Comment by xumix** [comment]
Sry, no

**Comment by JimBobSquarePants** [comment]
Fixed in v4.2.0
